logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/AssetRepository.kt

```diff
@@ -48,22 +84,28 @@ internal class AssetDataSource(
     }
 
     override suspend fun downloadPublicAsset(assetKey: String): Either<CoreFailure, ByteArray> = suspending {
-        wrapStorageRequest { assetDao.getAssetByKey(assetKey).firstOrNull() }
-            .coFold({
-                wrapApiRequest { assetApi.downloadAsset(assetKey, null) }
-                    .flatMap { assetData ->
-                        wrapStorageRequest { assetDao.insertAsset(assetMapper.fromUserAssetToDaoModel(assetKey, assetData)) }
-                            .map { assetData }
-                    }
-            }, {
-                Either.Right(it.rawData)
-            })
+        downloadAsset(assetKey, null)
```

Comment:
I think if we use named arguments in cases where the values of the parameters are not clear, would help the readability of the code.